src/components/ButtonPanel.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Button from './Button';
+
+function ButtonPanel() {
+  return (
+    <div>
+      <div>
+        <Button buttonName="AC" />
```

Comment:
- [ ] **[OPTIONAL]** Instead of hard-coding the button since they all share the same props, we can create multi-dimensional array to store button names and apply map to this array to create the button components.